docs/learn/learn-xcvm.md

```diff
@@ -187,6 +187,19 @@ Keep in mind that this location is specific to this interaction. The identities
 if this location was defined on another consensus system, such as Kusama. On other consensus
 systems, such as Ethereum, it won't be able to interpret it.
 
+### UniversalLocation in XCM
+
+While a `MultiLocation` may handle relative paths in _local_ consensus, a `UniversalLocation` refers
```

Comment:
No - a `MultiLocation` is just a relative location. It has no concept of local consensus and can trivially refer to systems outside of local consensus.

Reply:
This is still wrong. Why was the comment ignored??